archive/archive_request_ingest/src/test_archive_request_ingest.py

```diff
@@ -56,7 +54,7 @@ def test_sends_request_to_sns_with_callback(sns_client, topic_arn):
 def test_invalid_url_fails(sns_client, topic_arn):
     request = ingest_request('invalidUrl')
 
-    with pytest.raises(ValueError, match="\[BadRequest\] Unrecognised url scheme: invalid"):
+    with pytest.raises(BadRequestError, match="\[BadRequest\] Unrecognised url scheme: 'invalidUrl'"):
         request_ingest.main(event=request, sns_client=sns_client)
```

Comment:
Are the []s still needed?  It's a bit confusing because of the regexp, now it's all more consistent perhaps it can just be a prefix?